packages/types/src/company.ts

```diff
@@ -4,10 +4,10 @@ export const CompanySchema = z.object({
   id: z.string().uuid(),
   createdAt: z.date(),
   name: z.string().max(50),
-  description: z.string(),
+  description: z.string().nullable(),
   phone: z.string().nullable(),
-  email: z.string(),
-  website: z.string(),
+  email: z.string().nullable(),
+  website: z.string().nullable(),
```

Comment:
I think it's best to keep these required

Reply:
agreed